src/checkers/inference/model/ArithmeticConstraint.java

```diff
@@ -57,6 +57,16 @@ public static ArithmeticOperationKind fromTreeKind(Kind kind) {
                     return OR;
                 case XOR:
                     return XOR;
+                case PLUS_ASSIGNMENT:
```

Comment:
Can you add a comment somewhere, why the `_ASSIGNMENT` versions can be handled the same as the non-assignment versions?